pkg/controller/perconaservermongodb/backup.go

```diff
@@ -186,3 +185,29 @@ func (r *ReconcilePerconaServerMongoDB) isBackupRunning(cr *api.PerconaServerMon
 
 	return false, nil
 }
+
+func (r *ReconcilePerconaServerMongoDB) updatePBMConfig(cr *api.PerconaServerMongoDB) error {
+	isRestoring, err := r.isRestoreRunning(cr)
+	if err != nil {
+		return fmt.Errorf("checking if restore running on pbm update: %w", err)
+	}
+
+	if isRestoring {
+		return nil
+	}
+
+	pbm, err := backup.NewPBM(r.client, cr)
+	if err != nil {
+		return fmt.Errorf("create pbm object: %w", err)
+	}
+
+	var stg api.BackupStorageSpec // getting the first and only storage from map
```

Comment:
as mentioned above,
`If the user had one bucket and pitr enabled and now adds one more bucket, PITR continues to work, we throw an error in the operator logs`
This part will not work in case if user add new storage and we will select new bucket each time)

Reply:
This case is done at users own risk as discussed previously with @spron-in and @delgod